internal/future/statestore/redis.go

```diff
@@ -17,17 +17,21 @@ package statestore
 import (
 	"context"
 	"fmt"
+	"math"
 	"time"
 
+	"github.com/gogo/protobuf/proto"
 	"github.com/gomodule/redigo/redis"
 	"github.com/sirupsen/logrus"
+	"github.com/tidwall/gjson"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	"open-match.dev/open-match/internal/config"
 	"open-match.dev/open-match/internal/future/pb"
 )
 
 var (
+	redisTTL    = -1
```

Comment:
This global variable, it's not necessary.

Reply:
Done. Cached cfg on the redis backend object.